google-cloud-pubsub/lib/google/cloud/pubsub/async_publisher/batch.rb

```diff
@@ -255,6 +255,10 @@ def empty?
             end
           end
 
+          def total_message_bytes
```

Comment:
If you make this public, make sure you either document it (so customers know how to use it), or mark it as `# @private` so it doesn't show up in reference documentation.

Reply:
The whole Batch class itself is marked as @private and that should cover this case.